src/scene/graph-node.js

```diff
@@ -107,6 +108,23 @@ Object.assign(pc, function () {
         }
     });
 
+    /**
+     * @private
+     * @readonly
+     * @name pc.GraphNode#scale
+     * @description The world scale of the graph node. The scale will only be correct if the graph node
+     * in question is not skewed as a result of scales and rotations applied higher in the hierarchy.
+     * @type pc.Vec3
+     */
+    Object.defineProperty(GraphNode.prototype, 'scale', {
```

Comment:
Wouldn't getScale() be more in keeping with the API?

Reply:
Updated in deference to your awesome comment.